config.md

```diff
@@ -74,7 +74,7 @@ For all platform-specific configuration values, the scope defined below in the [
     * Windows: one mount destination MUST NOT be nested within another mount (e.g., c:\\foo and c:\\foo\\bar).
     * Solaris: corresponds to "dir" of the fs resource in [zonecfg(1M)][zonecfg.1m].
 * **`source`** (string, OPTIONAL) A device name, but can also be a directory name or a dummy.
-    Path values are either absolute or relative to the bundle.
+    Path values are either absolute path or relative path to the bundle.
```

Comment:
This seems redundant.  In the current wording, "absolute" and "relative" are adjectives describing the "Path values", and usage like "X path is absolute" or "X path is relative" are pretty common.